rosidl_cmake/cmake/rosidl_generate_interfaces.cmake

```diff
@@ -33,11 +33,14 @@ include(CMakeParseArguments)
 # :type DEPENDENCIES: list of strings
 # :param SKIP_INSTALL: if set skip installing the interface files
 # :type SKIP_INSTALL: option
+# :param LINT_INTERFACES: if set lint the interface files using
+#   the ``ament_lint`` package
```

Comment:
Since the macro itself does not perform any linting directly but only registers tests I think this should be reflected in the name of the option: e.g. `ADD_LINTER_TESTS`